src/integration-test/java/uk/gov/hmcts/juror/api/moj/controller/JurorManagementControllerITest.java

```diff
@@ -142,6 +143,40 @@ void createCheckInHappyPath() {
         validateAppearanceRecord(response);
     }
 
+    @Test
+    @DisplayName("POST addAttendanceDay() - happy path")
+    @Sql({"/db/mod/truncate.sql", "/db/jurormanagement/InitAddAttendanceDay.sql",
+        "/db/JurorExpenseControllerITest_expenseRates.sql"})
+    void addAttendanceDayHappyPath() {
```

Comment:
Needs tests for,

Unauthorised, Bad Payload, pool not found, user does not have access to pool etc

Reply:
Added

Reply:
Addded